app/src/processing/app/syntax/SketchTextAreaDefaultInputMap.java

```diff
@@ -67,8 +67,16 @@ public SketchTextAreaDefaultInputMap() {
 
       put(KeyStroke.getKeyStroke(KeyEvent.VK_A, ctrl), DefaultEditorKit.beginLineAction);
       put(KeyStroke.getKeyStroke(KeyEvent.VK_E, ctrl), DefaultEditorKit.endLineAction);
+      put(KeyStroke.getKeyStroke(KeyEvent.VK_N, ctrl), DefaultEditorKit.downAction);
+      put(KeyStroke.getKeyStroke(KeyEvent.VK_P, ctrl), DefaultEditorKit.upAction);
+      put(KeyStroke.getKeyStroke(KeyEvent.VK_B, ctrl), DefaultEditorKit.backwardAction);
+      put(KeyStroke.getKeyStroke(KeyEvent.VK_F, ctrl), DefaultEditorKit.forwardAction);
+      put(KeyStroke.getKeyStroke(KeyEvent.VK_D, ctrl), DefaultEditorKit.deleteNextCharAction);
+      put(KeyStroke.getKeyStroke(KeyEvent.VK_Y, ctrl), DefaultEditorKit.pasteAction);
+      put(KeyStroke.getKeyStroke(KeyEvent.VK_W, ctrl), DefaultEditorKit.cutAction);
```

Comment:
The 'cut' actions that here is set as Ctrl-W seems to be Ctrl-K instead.
From various sources I see that the mnemonics are:

Ctrl-K -> Kill (cut)
Ctrl-Y -> Yank (paste)

is that right?
All the other shortcuts works as expected.